alibi_detect/od/backends/torch/knn.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import torch
+
+
+class KNNTorch:
+    def score(X, x_ref, k, kernel=None):
```

Comment:
Personally prefer all lowercase as for the drift detectors. Also see comment on why the input args would be necessary since they are all already attributes of the method.